<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>576</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="baseSize">
    <size>
     <width>500</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPlainTextEdit" name="te_desxifrat">
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Text desxifrat</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Clau</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_clau">
          <property name="value">
           <number>11</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bt_xifrar">
        <property name="text">
         <string>Xifrar &gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_desxifrar">
        <property name="text">
         <string>&lt; Desxifrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_majuscules">
        <property name="text">
         <string>Maj√∫scules</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="te_xifrat">
      <property name="placeholderText">
       <string>Text xifrat</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
